sonatype.sbt

```diff
@@ -0,0 +1,22 @@
+// Your profile name of the sonatype account. The default is the same with the organization value
+sonatypeProfileName := "com.rallyhealth"
+
+// To sync with Maven central, you need to supply the following information:
+publishMavenStyle := true
+
+// publish to Maven Central
+ThisBuild / sonatypeCredentialHost := "s01.oss.sonatype.org"
+ThisBuild / publishTo := Some {
+  if (isSnapshot.value)
+    Resolver.url("Sonatype", url("https://s01.oss.sonatype.org/content/repositories/snapshots"))
+  else
+    Resolver.url("Sonatype", url("https://s01.oss.sonatype.org/content/repositories/releases"))
```

Comment:
HINT: It might make sense to rip out the common code

Reply:
I agree. I'm under a time crunch right now, but I plan to move some of this logic into various sbt plugins.